docs/tutorials/goldenpath.md

```diff
@@ -0,0 +1,497 @@
+{
+    private NetworkVariable<float> ServerNetworkVariable = new NetworkVariable<float>();
+    private NetworkVariable<float> ClientNetworkVariable = new NetworkVariable<float>();
+    private float last_t = 0.0f;
+
+    void Start()
+    {
+        ClientNetworkVariable.Settings.WritePermission = NetworkVariablePermission.OwnerOnly;
+        ClientNetworkVariable.Settings.ReadPermission = NetworkVariablePermission.ServerOnly;
+
+        if (IsServer)
+        {
+            ServerNetworkVariable.Value = 0.0f;
+            Debug.Log("Server's var initialized to: " + ServerNetworkVariable.Value);
+        }
+        else if (IsClient)
+        {
+            ClientNetworkVariable.Value = 0.0f;
+            Debug.Log("Client's var initialized to: " + ClientNetworkVariable.Value);
+        }
+    }
+
+    void Update()
+    {
+        var t_now = Time.time;
+        if (IsServer)
+        {
+            ServerNetworkVariable.Value = ServerNetworkVariable.Value + 0.1f;
+            if (t_now - last_t > 0.5f)
+            {
+                last_t = t_now;
+                Debug.Log("Server set its var to: " + ServerNetworkVariable.Value + ", has client var at: "  + 
+                    ClientNetworkVariable.Value);
+            }
+        }
+        else if (IsClient)
+        {
+            ClientNetworkVariable.Value = ClientNetworkVariable.Value + 0.1f;
+            if (t_now - last_t > 0.5f)
+            {
+                last_t = t_now;
+                Debug.Log("Client set its var to: " + ClientNetworkVariable.Value + ", has server var at: "  + 
+                    ServerNetworkVariable.Value);
+            }
+        }
+    }
+}
+```
+</details>
+
+1. Save your scene.
+
+<iframe src="https://www.youtube.com/embed/JsEVE6UaBIU?playlist=JsEVE6UaBIU&loop=1&&autoplay=0&controls=1&showinfo=0&mute=1"   width="854px"
+        height="480px" className="video-container" frameborder="0" position="relative" allow="accelerometer; autoplay; loop; playlist; clipboard-write; encrypted-media; gyroscope; picture-in-picture"  allowfullscreen=""></iframe>
+
+
+### Testing Server-controlled Network Variables 
+
+1. Select **File > Build and Run**. 
+1. Stop the player 
+1. Launch the client & server together in a terminal as shown in [Testing the command line helper](#testing-the-command-line-helper). 
+1. After a brief delay, the client and server will spawn.  
+1. You should expect to see this in the console, showing that the server and client are sharing the variable. 
+
+```
+Server's var initialized to: 0
+Client's var initialized to: 0
+Server set its var to: 0.1, has client var at: 0
+Client set its var to: 0.1, has server var at: 0.2
+Server set its var to: 3.099999, has client var at: 2.6
+Client set its var to: 3.099999, has server var at: 3.199999
+Server set its var to: 6.099997, has client var at: 5.599997
+```
+:::note
+Since the printing doesn't happen on every tick, the numbers won't match up perfectly.
```

Comment:
messages do generate on every tick but the printing to the log doesn't so they dont match up, I will clarify in the text.

:::note
Since the printing to the terminal  does not happen on every tick, the numbers will not match up perfectly.
:::